src/generateTasks.js

```diff
@@ -2,16 +2,19 @@
 
 const minimatch = require('minimatch')
 
+const readConfigOption = require('./readConfigOption')
+
 module.exports = function generateTasks(config, files) {
     const linters = config.linters !== undefined ? config.linters : config
     const resolve = file => files[file]
     return Object.keys(linters)
         .map((pattern) => {
             const commands = linters[pattern]
-            const filter = minimatch.filter(pattern, {
+            const globOptions = readConfigOption(config, 'globOptions', {})
+            const filter = minimatch.filter(pattern, Object.assign({
                 matchBase: true,
```

Comment:
@okonet that's not the same. According to `readConfigOption` the default is only used if nothing is supplied. If a use sets `{ nocase: true }` in your suggestion it will also remove `matchBase: true, dot: true` because it will never use the default.

When `Object.assign()` is used later the user can override any keys without affecting the defaults (or override the defaults)